packages/next/errors.json

```diff
@@ -667,11 +667,9 @@
   "666": "Turbopack builds are only available in canary builds of Next.js.",
   "667": "receiveExpiredTags is deprecated, and not expected to be called.",
   "668": "Internal Next.js error: Router action dispatched before initialization.",
-  "669": "Invariant: --turbopack is set but the build used Webpack",
-  "670": "Invariant: --turbopack is not set but the build used Turbopack. Add --turbopack to \"next start\".",
```

Comment:
`errors.json` needs to be a historic document. Otherwise telemetry of old events will no longer match up.

Reply:
Sounds like we need a CI check to watch for this

Reply:
Reset to `canary` and just did normal build so if it's out of sync now I don't think our setup is allowing for persisting stale errors.

Reply:
We do check in CI if messages are up-to-date. But we can't really check if you truly are removing a message. We still want the option to amend existing messages. So a check that tries to guess if you actually remove a message vs amend a message would have a lot of false-positives.